certbot_external_auth/plugin.py

```diff
@@ -724,6 +727,26 @@ def _call_handler(self, command, *args, **kwargs):
     # Helper methods & UI
     #
 
+    def _json_sanitize_dict(self, dictionary):
+        for key, val in list(dictionary.items()):
+            # Not highly effecient, would be neater to clean up FIELD_TOKEN.
+            # But if any of the others turn to bytes in the future, this will solve it:
+            if type(key) == bytes:
+                del dictionary[key]
+                key = key.decode('UTF-8')
+                dictionary[key] = val
+
+            if type(val) == bytes:
+                dictionary[key] = val.decode('UTF-8')
+            elif type(val) in (list, tuple):
```

Comment:
We could remove the list support for now and add a comment about expanding into a recursive function if needed in the future?